Spatial_searching/include/CGAL/Splitters.h

```diff
@@ -19,6 +19,7 @@
 
 #include <CGAL/license/Spatial_searching.h>
 
+#include <CGAL/basic.h>
```

Comment:
Please avoid `<CGAL/basic.h>`. That header is a plague.

It seems you need `CGAL::is_zero`, so:

```suggestion
#include <CGAL/number_utils.h>
```